premailer/tests/test_premailer.py

```diff
@@ -505,7 +503,7 @@ def test_base_url_with_path(self):
         </body>
         </html>'''
 
-        p = Premailer(html, base_url='http://kungfupeople.com/base',
+        p = Premailer(html, base_url='http://kungfupeople.com/base/',
```

Comment:
I don't think there is. But the result should be identical to what `urljoin` does.